sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLKitDAO.java

```diff
@@ -187,6 +183,7 @@ public int count() throws IOException {
     return template.query(KITS_SELECT_BY_RELATED_EXPERIMENT, new Object[] { experimentId }, new KitMapper());
   }
 
+  // There is currently no link table between kit and library.
```

Comment:
This method isn't used anywhere and doesn't really make sense since. Could be removed